test/test_kiwrap.py

```diff
@@ -185,7 +197,7 @@ def test_kiwrap_check_repokey_no_trusted_key_and_no_key_path(self, capsys: Captu
             try:
                 KiwiParent_instance._check_repokey(repodata, reponame)
             except SystemExit as s:
-                assert s.args == ("key file path wrong for repository test",)
+                assert s.args == ("Wrong key file path for repository test",)
```

Comment:
Don't use `try/except`, they should be mocked instead.

Reply:
i don't get your point , i've updated a string message

Reply:
https://github.com/Pytest-with-Eric/pytest-magicmock-raise-exception-example/